### PR TITLE
Check if gnome-icon-theme is installed

### PR DESCRIPTION
**Pull request by atodorov**
Just minimize the false negatives when this is installed.

### CODE REVIEW

**Comment by atodorov** [comment]
also checks for gnome-icon-theme-symbolic

File: tests/glade/run_glade_tests.sh [hunk 1]
```diff
@@ -5,6 +5,18 @@ if ! type parallel 2>&1 > /dev/null; then
     exit 99
 fi
 
+if ! rpm -q gnome-icon-theme 2>&1 > /dev/null; then
```
Comment (vpodzime) [inline tests/glade/run_glade_tests.sh]: I think ``! rpm -q gnome-icon-theme &> /dev/null;`` would be better here and few lines below too.

**Comment by vpodzime** [comment]
Other than the minor comment above this looks good to me.

**Comment by clumens** [comment]
It looks like we'd need this on master too, right?

**Comment by atodorov** [comment]
> It looks like we'd need this on master too, right?

Yes

**Comment by clumens** [comment]
Pushed to both master and rhel7-branch.  Thanks.

**Pull request closed without merging**